amazon-kinesis-client.iml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like the iml file accidentally got pushed :)

Reply:
Eh... MB